games-emulation/pcsx2/files/pcsx2-1.3.1-custom-cflags.patch

```diff
@@ -0,0 +1,347 @@
+From b9be633dfe3c086c2cd9cb6ebe7fad9a1bb18ff3 Mon Sep 17 00:00:00 2001
+From: nE0sIghT <ykonotopov@gmail.com>
+Date: Sat, 18 Apr 2015 23:27:55 +0300
+Subject: [PATCH] Stripped optimization flags.
+ https://github.com/nE0sIghT/vortex-overlay/issues/4
```

Comment:
OMG. Hmm… could you maybe try if you could override that all with `-DOptimizationFlags=`?